src/main/java/org/isf/sms/providers/gsm/GSMGatewayService.java

```diff
@@ -125,97 +131,88 @@ public boolean initialize() {
 			}
 		} else {
 			LOGGER.error("COM PORT not found ({})!!!", port);
- 		}
+		}
 		return connected;
 	}
 
 	@Override
-	public boolean sendSMS(Sms sms, boolean debug) {
+	public boolean sendSMS(Sms sms) {
```

Comment:
the `debug` flag was useful for testing mocking the actual SMS sending, is there another way to do it that I don't see in this PR's code?

Reply:
I restored the previous code.